README.md

```diff
@@ -200,11 +200,14 @@ class Program
 }
 ```
 
-### Working with Apache Avro
+### Schema Registry Integration
 
-The `Confluent.SchemaRegistry.Serdes` nuget package provides an Avro serializer and deserializer that integrate with [Confluent
-Schema Registry](https://docs.confluent.io/current/schema-registry/docs/index.html). The `Confluent.SchemaRegistry` 
-nuget package provides a client for interfacing with Schema Registry's REST API.
+The three "Serdes" packages provide serializers and deserializers Avro, Protobuf and JSON with [Confluent Schema Registry](https://docs.confluent.io/current/schema-registry/docs/index.html) integration. The `Confluent.SchemaRegistry` nuget package provides a client for interfacing with
```

Comment:
seems to be punctuation or a word missing after "deserializers".

Reply:
well spotted, thanks.